addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1266,6 +1270,86 @@ options.registry.parallax = options.Class.extend({
     onMove: function () {
         this._refreshPublicWidgets();
     },
+
+    //--------------------------------------------------------------------------
+    // Options
+    //--------------------------------------------------------------------------
+
+    /**
+     * Build/remove parallax.
+     *
+     * @see this.selectClass for parameters
+     */
+    parallax: function (previewMode, widgetValue, params) {
+        const isParallax = widgetValue === 'true';
+        this.$target.toggleClass('parallax', isParallax);
+        if (isParallax) {
+            this._initParallax();
+        } else {
```

Comment:
I am not sure how this could work. `_initParallax` calls background option update before the parallax is actually build (as the animation will only be refreshed automatically at the end of the `parallax` option method here. Should not you call refreshPublicWidget just after toggling the class parallax?

This will also allow you to explain that the animation is responsible for building the parallax, which will make the code clearer.

Reply:
> This will also allow you to explain that the animation is responsible for building the parallax, which will make the code clearer.

So explain it... :)